hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -1230,22 +1246,29 @@ solHandlers (CHandlers hs) =
 divup :: Integer -> Integer -> Integer
 divup x y = ceiling $ (fromIntegral x :: Double) / (fromIntegral y)
 
-solBytesSplit :: Integer -> (Integer -> Integer -> a) -> [a]
-solBytesSplit sz f = map go [0 .. lastOne]
+maxStringLen :: Integer
+maxStringLen = 32
```

Comment:
This is a deceptive name... it is like the size of the chunks... so `byteChunkSize`?